orttraining/orttraining/training_ops/cpu/controlflow/ort_tasks.h

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+#include "core/framework/ml_value.h"
+
+#include <mutex>
+#include <future>
+
+
+namespace onnxruntime {
+namespace contrib {
+
+typedef std::pair<Status, std::vector<OrtValue>> ForwardReturnType;
+typedef std::pair<bool, std::vector<OrtValue>> BackwardReturnType;
+
+class OrtTasks final {
```

Comment:
>class OrtTasks [](start = 0, length = 14)

class and member function documentation would be useful

Reply:
As I mentioned above, we are working on a new design that will remove the need for OrtTask class, so I will skip documenting this class.  

This is the PR in progress: #7077